index.html

```diff
@@ -7,6 +7,33 @@
   <link rel="stylesheet" href="./css/styles.css">
 </head>
 <body>
-  Starting html template. Build from here.
+    <header>
+      <div class="logo">
+        <img src="img/logo.png">
+      </div>
+      <div class="main">
+      <ul class="container-1">
```

Comment:
Use `<nav><a>Link</a></nav>` instead of `<ul>` and `<li>`. This will add up to the Semantics of the page. and also you will have "inline" links by default.